tests/e2e/utils/check-verification-code.ts

```diff
@@ -7,41 +7,48 @@ const localizedStrings = {
   sudo: locales.en["/account/security/sudo"],
 };
 
+async function getCode(emailAddress: string): Promise<string> {
```

Comment:
This function takes the last email sent to the specified address, gets the code and delete the email.

Reply:
Can we rename this to `getValidationCode`? I was confused about the naming in the above tests 😄.